packages/fiori/src/DynamicPage.ts

```diff
@@ -226,6 +226,13 @@ class DynamicPage extends UI5Element {
 	}
 
 	onBeforeRendering() {
+		if (!this.mediaRange) {
+			this.mediaRange = MediaRange.getCurrentRange(
```

Comment:
Can we improve `updateMediaRange` function to work before rendering of the DOM and call it here?

Reply:
Overall why do we need media range? Is it only related to the responsive paddings? If yes, probably we can remove it and to replace resize observer with content queries. We also have enhancement request to replace all resize observer that are added with this purpose (https://github.com/SAP/ui5-webcomponents/issues/7666)